tokens/schemas/token-group.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Token group schema",
+  "type": "object",
+  "additionalProperties": {
+    "type": "string"
+  }
+}
```

Comment:
If folks haven't worked with JSON Schema this is validating that our tokens are objects with a depth of 1 whose values are strings. Here is an example of the error thrown if we violate this constraint:
![Screen Shot 2021-12-08 at 3 13 41 PM](https://user-images.githubusercontent.com/32409546/145310393-431c8d1a-11de-49a0-b049-a4901dcd32de.png)